README.md

```diff
@@ -19,23 +19,23 @@ const isAvailable = await gcpMetadata.isAvailable();
 
 #### Access all metadata
 ```js
-const res = await gcpMetadata.instance();
-console.log(res.data); // ... All metadata properties
+const data = await gcpMetadata.instance();
+console.log(data); // ... All metadata properties
 ```
 
 #### Access specific properties
 ```js
-const res = await gcpMetadata.instance('hostname');
-console.log(res.data) // ...All metadata properties
+const data = await gcpMetadata.instance('hostname');
+console.log(data) // ...All metadata properties
```

Comment:
You didn't break this, but the comment seems wrong. It should be `// instance hostname`.